daprdocs/content/en/reference/api/error_codes.md

```diff
@@ -7,43 +7,150 @@ weight: 1400
 ---
 
 For http calls made to Dapr runtime, when an error is encountered, an error json is returned in http response body. The json contains an error code and an descriptive error message, e.g.
```

Comment:
@jake-engelberg this is HTTP and gRPC?

Reply:
Yes, it is both and we'll update the wording soon to reflect the auto generated chart with both codes which at the moment sometimes differ:

https://github.com/dapr/dapr/pull/8346